support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -950,6 +946,13 @@ form {
 
 /* Server side rendering test loading message*/
 
+
+.gu-content--contribution-form--placeholder {
```

Comment:
Since the georedirect happens before anything is rendered, why do we need to hide this?

Reply:
_Marked as resolved_